lib/swaggerValidator/modelValidator.ts

```diff
@@ -938,6 +941,34 @@ export class SwaggerExampleValidator {
         )
       );
     }
+    if (
```

Comment:
Can you combine this if with above if section like: 
at line#925, if (provider?.type === "resource-manager" || (provider?.type === "data-plane" &&
      (headers["Operation-Id"] === undefined || headers["Operation-Id"] === "") &&
      (headers["operation-id"] === undefined || headers["operation-id"] === "") &&
      (headers["Operation-Location"] === undefined || headers["Operation-Location"] === "") &&
      (headers["operation-location"] === undefined || headers["operation-location"] === ""))) {
      -- push the error
}